recipes/ffmpeg/all/conanfile.py

```diff
@@ -0,0 +1,472 @@
+            opt_enable_disable("libzmq", self.options.with_zeromq),
+            opt_enable_disable("sdl2", self.options.with_sdl),
+            opt_enable_disable("libx264", self.options.with_libx264),
+            opt_enable_disable("libx265", self.options.with_libx265),
+            opt_enable_disable("libvpx", self.options.with_libvpx),
+            opt_enable_disable("libmp3lame", self.options.with_libmp3lame),
+            opt_enable_disable("libfdk-aac", self.options.with_libfdk_aac),
+            opt_enable_disable("libwebp", self.options.with_libwebp),
+            opt_enable_disable("openssl", self.options.with_ssl == "openssl"),
+            opt_enable_disable("alsa", self.options.get_safe("with_libalsa")),
+            opt_enable_disable("libpulse", self.options.get_safe("with_pulse")),
+            opt_enable_disable("vaapi", self.options.get_safe("with_vaapi")),
+            opt_enable_disable("vdpau", self.options.get_safe("with_vdpau")),
+            opt_enable_disable("libxcb", self.options.get_safe("with_xcb")),
+            opt_enable_disable("libxcb-shm", self.options.get_safe("with_xcb")),
+            opt_enable_disable("libxcb-shape", self.options.get_safe("with_xcb")),
+            opt_enable_disable("libxcb-xfixes", self.options.get_safe("with_xcb")),
+            opt_enable_disable("appkit", self.options.get_safe("with_appkit")),
+            opt_enable_disable("avfoundation", self.options.get_safe("with_avfoundation")),
+            opt_enable_disable("coreimage", self.options.get_safe("with_coreimage")),
+            opt_enable_disable("audiotoolbox", self.options.get_safe("with_audiotoolbox")),
+            opt_enable_disable("videotoolbox", self.options.get_safe("with_videotoolbox")),
+            opt_enable_disable("securetransport", self.options.with_ssl == "securetransport"),
+            "--disable-cuda",  # FIXME: CUDA support
+            "--disable-cuvid",  # FIXME: CUVID support
+            # Licenses
+            opt_enable_disable("nonfree", self.options.with_libfdk_aac),
+            opt_enable_disable("gpl", self.options.with_libx264 or self.options.with_libx265 or self.options.with_postproc)
+        ]
+        if self._arch:
+            args.append("--arch={}".format(self._arch))
+        if self.settings.build_type == "Debug":
+            args.extend([
+                "--disable-optimizations",
+                "--disable-mmx",
+                "--disable-stripping",
+                "--enable-debug",
+            ])
+        # since ffmpeg"s build system ignores CC and CXX
+        if tools.get_env("AS"):
+            args.append("--as={}".format(tools.get_env("AS")))
+        if tools.get_env("CC"):
+            args.append("--cc={}".format(tools.get_env("CC")))
+        if tools.get_env("CXX"):
+            args.append("--cxx=".format(tools.get_env("CXX")))
+        if self.settings.compiler == "Visual Studio":
+            args.append("--pkg-config={}".format(tools.get_env("PKG_CONFIG")))
```

Comment:
perhaps, it was filling an empty string for Linux? anyway, I guess we can assume linux systems have pkg-config installed for the first iteration